integration-tests/elytron-security-oauth2/src/main/java/io/quarkus/it/elytron/oauth2/MyResource.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.it.elytron.oauth2;
+
```

Comment:
There has to be a native version it (not 100% sure it will be critical for this initial PR but I guess it might be)

Reply:
done